xpdacq/beamtime.py

```diff
@@ -86,43 +89,45 @@ def ct(dets, exposure, *, md=None):
                         'sp_computed_exposure': computed_exposure,
                         'sp_type': 'ct',
                         # need a name that shows all parameters values
-                        #'sp_name': 'ct_<exposure_time>',
+                        # 'sp_name': 'ct_<exposure_time>',
                         'sp_uid': str(uuid.uuid4()),
                         'sp_plan_name': 'ct'})
-    plan = bp.count([glbl.area_det], md = _md)
+    plan = bp.count([glbl.area_det], md=_md)
     plan = bp.subs_wrapper(plan, LiveTable([glbl.area_det]))
     yield from plan
 
 
-def Tramp(dets, exposure, Tstart, Tstop, Tstep, *, md=None):
+def Tramp(dets, exposure, tstart, tstop, tstep, *, md=None):
```

Comment:
+1 for leaving it as is. See "good reasons to ignore a particular guideline" section of PEP8.